src/editor/components/guided-modal-tour/editor.scss

```
@@ -0,0 +1,236 @@
+	background-color: transparent !important;
+	pointer-events: none;
+}
+
+.interact-tour-modal {
+	pointer-events: all;
+	position: absolute;
+	--offset-x: 0px;
+	--offset-y: 0px;
+	--left: 50%;
+	--top: 50%;
+	left: var(--left);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)